compiler/damlc/daml-stdlib-src/DA/Internal/Desugar.daml

```diff
@@ -8,10 +8,11 @@ daml 1.2
 -- | Automatically imported qualified in every module.
 module DA.Internal.Desugar (
     concat,
-    Template(ensure, signatory, observer, agreement),
-    TemplateKey(key, maintainer),
-    Choice(consuming, choiceController, choice), preconsuming, nonconsuming, postconsuming, NoEvent(..),
-    IsParties(toParties),
+    module DA.Internal.Template,
```

Comment:
What's the reasoning behind this change?

Reply:
The goal was to export everything in `DA.Internal.Template`. Is there anything there we want to hide?

Reply:
Is there anything that was hidden before? If not, we can do it like you propose.

Reply:
I think everything needs be exported now.